### PR TITLE
Add Grav, MediaWiki and DokuWiki quick-install scripts

### PR DESCRIPTION
**Pull request by PsychotherapistSam**
I will keep MediaWiki & DokuWiki up-to-date, Grav is always up-to-date.

### CODE REVIEW

File: web/src/app/WebApp/Installers/DokuWiki/DokuWikiSetup.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		  . "--data 'd[policy]=" . substr($options['initial_ACL_policy'], 0, 1) . "' "
+		  . "--data 'd[license]=" . explode(":", $options['content_license'])[0] . "' "
+		  . "--data submit=";
+
+		exec($cmd, $msg, $code);
+
+		// remove temp folder
+		$this->cleanup();
+
+		return ($code === 0);
```
Comment (jaapmarcus) [inline web/src/app/WebApp/Installers/DokuWiki/DokuWikiSetup.php:101]: If SSL is not enabled It doesn't work

File: web/src/app/WebApp/Installers/MediaWiki/MediaWikiSetup.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		'resources' => [
+			'archive'  => [ 'src' => 'https://releases.wikimedia.org/mediawiki/1.36/mediawiki-1.36.1.zip' ],
+		], 
+	];
+	
+	public function install(array $options = null)
+	{
+		parent::install($options);
+
+		$webAddresss = "https://" . $this->domain;
```
Comment (jaapmarcus) [inline web/src/app/WebApp/Installers/MediaWiki/MediaWikiSetup.php]: Same here as docuwiki

**Review by jaapmarcus — COMMENTED** [review]
GravCMS didn't work at all we might be able to replace it with 

https://github.com/jaapmarcus/hestiacp/tree/feature/garvcms

Docuwiki seems to be working fine except a ssl in this care required we should be able add  a check if SSL is enabled or not see Prestashop

Mediawiki is missing the NGINX  template for Nginx only + Didn't work with SSL disabled

File: web/src/app/WebApp/Installers/Grav/GravSetup.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+		'group' => 'cms',
+		'enabled' => true,
+		'version' => 'latest',
+		'thumbnail' => 'grav-symbol.svg'
+	];
+	
+	protected $appname = 'grav';
+	
+	protected $config = [
+		'form' => [],
```
[the remark this answers is not in the document]
Comment (PsychotherapistSam) [reply web/src/app/WebApp/Installers/Grav/GravSetup.php]: I had a temporary field there, as Grav doesn't need one. Should I just add one with a lable to ignore it?

File: web/src/app/WebApp/Installers/DokuWiki/DokuWikiSetup.php (hunk 1 above)
Comment (PsychotherapistSam) [reply web/src/app/WebApp/Installers/DokuWiki/DokuWikiSetup.php:101]: I'll add a check whether ssl is enabled and just perform the curl request using --insecure on http

**Comment by PsychotherapistSam** [comment]
The SSL Issues should be resolved now. Also the PrestaShop SSL check doesn't seem to work, I modified it a bit and the version implemented for DokuWiki and MediaWiki seems to work.
Also I enabled the .htaccess file for DokuWiki

**Comment by jaapmarcus** [comment]
Made some changes to GravCMS + Added templates for it.

For Docuwiki I have removed the install.php after install.

**Pull request merged**